CHANGELOG.md

```diff
@@ -1,160 +1,228 @@
-[//]: # (START/LATEST)
+[//]: # "START/LATEST"
+
 # Latest
 
 ## Features
-  * A user-friendly description of a new feature. {issue-number}
+
+- A user-friendly description of a new feature. {issue-number}
+
+## Fixes
+
+- A user-friendly description of a fix. {issue-number}
+
+## Security
+
+- A user-friendly description of a security fix. {issue-number}
+
+---
+
+[//]: # "START/v2.0.0"
+
+# v2.0.0
+
+## Features
+
+- Added support for `Secure Note` items. {149}
+- Added `note_value` attribute representing a 1Password Item's `notes` field. {57}
 
 ## Fixes
- * A user-friendly description of a fix. {issue-number}
+
+- The data handling is more robust, making it less prone to errors and inconsistencies. {157,146}
+- CLI and Connect clients now have a more consistent behavior.
+- Fields of type `OTP` are better handled when user provides a custom ID for them. Terraform will throw an error if the custom ID doesn't have the `TOTP_` prefix, which is required for this field type.
+- The values that are generated will only show in the plan to be recomputed when the recipe is changed or the value is explicitly set.
+- When fetching Database items from 1Password, the `server` field (previously known as `hostname`) will populate the Terraform `hostname` attribute. This ensures that the data from new Database items is mapped as expected. {76}
+- Vault description is now fetched when getting a vault from 1Password by name and the provider was configured to use the CLI client.
+- Generated values (using a recipe) are now regenerated when the recipe is changed.
```

Comment:
Do we know the links to the issues/PR related to those fixes?

Reply:
There is no linked issue for this, it was discovered to be a problem during development but was fixed by nature of this upgrade